### DISCUSSION TITLE
How to update our hosted website when chirpy-startup theme comes with updates?

### DISCUSSION THREAD

**Discussion by anonfaded** [cotes2020/jekyll-theme-chirpy#1368] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I hv used the chirpy starter template but i did some customization like i added footer and sidebar folders where i edited the theme files, and now when the official theme has come with some updates on their GitHub repo and i want to get that updates on my own website too, how to get that? if i just sync that manually by downloading files and git pushing to my website repo will it also overwrite my customized theme? 

### Operations you have already tried

Haven't tried yet

**Comment by joshjohanning** [comment]
I use the fork method, but as an example this is what I do to upgrade the theme (using `git cherry-pick` and monitoring merge conflicts)

https://github.com/joshjohanning/joshjohanning.github.io/?tab=readme-ov-file#upgrading-the-theme

**Comment by kungfux** [comment, marked as answer]
Here is a wiki on [How to upgrade](https://github.com/cotes2020/jekyll-theme-chirpy/wiki/Upgrade-Guide) to the latest when you use `chirpy-starter` method.
